__tests__/AuthScreen-test.tsx

```diff
@@ -13,22 +14,24 @@ jest.mock('react-redux', () => {
   };
 });
 
-it.skip('AuthScreen is rendered', () => {
-  render(<AuthScreen />);
+jest.mock('react-native/Libraries/Linking/Linking', () => ({
+  openURL: jest.fn(() => Promise.resolve('mockResolve')),
+  getInitialURL: jest.fn(() => Promise.resolve('mockResolve')),
+  addEventListener: jest.fn(),
+}));
+
+it('AuthScreen is rendered', () => {
+  customRenderWithProvider(AuthScreen);
   screen.getByText(/welcome to/i);
   screen.getByText(/real dev squad/i);
 });
 
-it.skip('Clicking on Sign in with Github shows a toast', async () => {
-  render(
-    <>
-      <AuthScreen />
-      <Toast />
-    </>,
-  );
+it('Clicking on Sign in with Github opens browser', async () => {
+  customRenderWithProvider(AuthScreen);
+
   const githubSignInBtn = screen.getByText(Strings.SIGN_IN_BUTTON_TEXT);
   fireEvent.press(githubSignInBtn);
-  screen.getByText(/Sign in with GitHub coming soon/i);
+  expect(Linking.openURL).toHaveBeenCalledTimes(1);
```

Comment:
Can you change `toHaveBeenCalledTimes` with `toHaveBeenCalledWith` and ensure that this has been called with the correct link

Reply:
updated